charts/posthog/values.yaml

```diff
@@ -301,6 +301,9 @@ worker:
   # -- Count of worker pods to run. This setting is ignored if `worker.hpa.enabled` is set to `true`.
   replicacount: 1
 
+  workers:
```

Comment:
Perhaps we want "consumers" just to avoid the multipe use of the word "workers"?

`Our "worker" chart deploys "workers" each of which can have multiple celery "workers"` 😅

Reply:
That's better, yes!